components/Link/index.js

```diff
@@ -181,19 +172,26 @@ const Link = ({
 			{isLinkOpen && (
 				<Popover
 					position="top center"
-					onClose={() => setIsLinkOpen(false)}
```

Comment:
This was redundant since the `onClose` callback is only invoked when the popover gets unmounted